### PR TITLE
Add erd1qqqqqqqqqqqqqpgq9l2rc6wx6929npnn079z7tst4u5zlmaylzmsqt5n3h.json & update erd1qqqqqqqqqqqqqpgqgkjaenrw23wnp9cssfeplpau3dmkcwt6lzms5x7pqd.json

### PR DESCRIPTION
**Pull request by EdiMarian**

### CODE REVIEW

**Comment by EdiMarian** [comment]
{
  "address": "erd1kwv75jn5nu0z0avcvplfpq0r9x28y8ym4ypf8w99jstqxaxslzms6qjjd2",
  "message": "0x416464206572643171717171717171717171717171706771396c327263367778363932396e706e6e3037397a377473743475357a6c6d61796c7a6d737174356e33682e6a736f6e202620757064617465206572643171717171717171717171717171706771676b6a61656e72773233776e70396373736665706c70617533646d6b637774366c7a6d733578377071642e6a736f6e",
  "signature": "0x1244be9dd41c60e5c2d65581366efc72196a2b7e75cc0fcce6925913d638dc55f9910f9a777513cecea64fe7e41e0d67673d0e41501cad606cf3e6d6778b490b",
  "version": 1,
  "signer": "ErdJS"
}

File: accounts/erd1qqqqqqqqqqqqqpgq9l2rc6wx6929npnn079z7tst4u5zlmaylzmsqt5n3h.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  "name": "MultiFights: NFT Players Staking",
+  "description": "",
+  "social": {
+    "website": "https://www.multi-fights.com",
+    "blog": "https://medium.com/@multifights",
+    "twitter": "https://twitter.com/MultiFights",
+    "whitepaper": "https://whitepaper.multi-fights.com/overview/introduction",
+    "discord": "https://discord.gg/du8urb6pd6"
+  },
+  "tags": ["multifights", "NFT", "staking"]
```
Comment (cfaur09) [inline accounts/erd1qqqqqqqqqqqqqpgq9l2rc6wx6929npnn079z7tst4u5zlmaylzmsqt5n3h.json:11]: use lowercase for NFT

File: accounts/erd1qqqqqqqqqqqqqpgqgkjaenrw23wnp9cssfeplpau3dmkcwt6lzms5x7pqd.json [hunk 2]
```diff
@@ -8,9 +8,5 @@
...
     "discord": "https://discord.gg/du8urb6pd6"
   },
-  "tags": [
-    "multifights",
-    "NFT",
-    "staking"
-  ]
-}
\ No newline at end of file
+  "tags": ["multifights", "NFT", "staking"]
```
Comment (cfaur09) [inline accounts/erd1qqqqqqqqqqqqqpgqgkjaenrw23wnp9cssfeplpau3dmkcwt6lzms5x7pqd.json:11]: use lowercase for NFT

File: accounts/erd1qqqqqqqqqqqqqpgq9l2rc6wx6929npnn079z7tst4u5zlmaylzmsqt5n3h.json (hunk 1 above)
Comment (cfaur09) [reply accounts/erd1qqqqqqqqqqqqqpgq9l2rc6wx6929npnn079z7tst4u5zlmaylzmsqt5n3h.json:11]: also use 
"tags": 
["multifights",
 "nft",
 "staking"
]

instead
"tags": ["multifights", "NFT", "staking"]

File: accounts/erd1qqqqqqqqqqqqqpgqgkjaenrw23wnp9cssfeplpau3dmkcwt6lzms5x7pqd.json (hunk 2 above)
Comment (EdiMarian) [reply accounts/erd1qqqqqqqqqqqqqpgqgkjaenrw23wnp9cssfeplpau3dmkcwt6lzms5x7pqd.json:11]: :+1:  @cfaur09

**Review by paulmultiversx — APPROVED** [review]

**Review by paulmultiversx — CHANGES_REQUESTED** [review]
Hi, @EdiMarian.

We have a limitation at only 1 asset / PR so please split it in separate PRs.

Also, don't forget to paste in a comment the signature for the latest commit sha which must be signed with the owner wallet address.

Feel free to use the Sign Message feature from the Utils dApp and paste the signature in a comment for each PR: https://utils.multiversx.com/

Thanks a lot!

**Comment by EdiMarian** [comment]
{
  "address": "erd1kwv75jn5nu0z0avcvplfpq0r9x28y8ym4ypf8w99jstqxaxslzms6qjjd2",
  "message": "0x44656c657465206163636f756e74732f6572643171717171717171717171717171706771676b6a61656e72773233776e70396373736665706c70617533646d6b637774366c7a6d733578377071642e6a736f6e",
  "signature": "0xfb2d57e2e3dc9fb6923e3ea408eef4243a1d5ff3184d68aa2e15770b01944678b1df23996b6f1700489467debff54303b3eb7b0854b9861eb75728da852b3903",
  "version": 1,
  "signer": "ErdJS"
}

**Review by paulmultiversx — DISMISSED** [review]

**Comment by EdiMarian** [comment]
Sorry, I signed with the commit name :)) @paulmultiversx 
{
  "address": "erd1kwv75jn5nu0z0avcvplfpq0r9x28y8ym4ypf8w99jstqxaxslzms6qjjd2",
  "message": "0x39343134636561643239646364316434623434613964323732313532626231646631316238353937",
  "signature": "0x3816d9b8eb60a31cd5ecd89a46201c67c08a27da5429961a95cc2bedd257aed2d0351c5e7a15d56f8147c9d1cf754f0a8cf6f05da8a292e034705c602e233700",
  "version": 1,
  "signer": "ErdJS"
}

**Comment by paulmultiversx** [comment]
Yes but you need only 1 account / PR. You still have 2 as it seems.

**Review by paulmultiversx — APPROVED** [review]

**Pull request merged**